editor/src/components/custom-code/code-file.ts

```diff
@@ -124,10 +124,18 @@ export function componentInfo(
   }
 }
 
+export type PlaceholderSpec =
+  // a placeholder that inserts a span that wraps `contents`. This will be
+  // editable with the text editor
+  | { type: 'text'; contents: string }
+  // inserts a `div` with the specified width and height
+  | { type: 'spacer'; width: number; height: number }
```

Comment:
We also want one which completely fills the available space

Reply:
added it in https://github.com/concrete-utopia/utopia/pull/5211/commits/298cef9a97a3b6588a485b7595070a1d1f75349a